src/backend/distributed/operations/shard_rebalancer.c

```diff
@@ -1975,10 +1973,8 @@ RebalanceTableShardsBackground(RebalanceOptions *options, Oid shardReplicationMo
 			(errmsg("Scheduled %d moves as job %ld",
 					list_length(placementUpdateList), jobId),
 			 errdetail("Rebalance scheduled as background job"),
-			 errhint("To monitor progress, run: "
-					 "SELECT * FROM pg_dist_background_task WHERE job_id = %ld ORDER BY "
-					 "task_id ASC; or SELECT * FROM get_rebalance_progress();",
-					 jobId)));
+			 errhint("To monitor progress, run: SELECT * FROM "
+					 "get_rebalance_status();")));
```

Comment:
The design docs suggested the query `SELECT details FROM get_rebalance_status();` here. However, I like this more because now the user can also see when the operation started, the current state of the rebalance job, its id, etc. Some of this information may be useful to the operator.

Reply:
sounds good to me.